### ISSUE TITLE
[Accessibility] [Ensures <img> elements have alternate text or a role of none or presentation (img)]

### ISSUE THREAD

**Issue by GayatriDeviP** [microsoft/AdaptiveCards#5610]
# Platform
- [ ] .NET HTML

# Author or host
D365 Sales

# Version of SDK
latest

# Details
For all the images in adaptive cards, there is an issue with accessibility as role of none or presentation is missing.

Repro: 
1. open any of sample adaptive card and place images beside the button or any where.
2. Install "Accessibility Insights for Web"(https://accessibilityinsights.io/docs/en/web/overview/) and add it to chrome or edge 
2. Run "Accessibility Insights for Web" and check the issue on image.

screenshot for reference:
![image](https://user-images.githubusercontent.com/58205918/113268486-0dbc0200-92f5-11eb-8de1-3a11314e0ec3.png)

**Comment by jonmill** [comment]
hi @GayatriDeviP - thanks for the issue; we are not currently investing in the .NET HTML renderer at this time
